data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/S3SourceConfig.java

```diff
@@ -38,13 +41,15 @@ public class S3SourceConfig {
     @Valid
     private AwsAuthenticationOptions awsAuthenticationOptions;
 
-    @JsonProperty("thread_count")
-    @Min(0)
-    private int threadCount;
-
     @JsonProperty("on_error")
     private OnErrorOption onErrorOption = OnErrorOption.RETAIN_MESSAGES;
 
+    @JsonProperty("request_timeout")
```

Comment:
I think we should give this a name like `buffer_timeout`. This isn't a timeout for any requests, just for putting in the Data Prepper buffer.